src/couchapps/T0Request/updates/updaterequest.js

```diff
@@ -36,11 +36,11 @@ function(doc, req)
     function updateTransition(newStatus) {
     	
     	if (allowedStates.indexOf(newStatus) === -1) {
-    		return "not allowed state " + newStatus;
+    		return "Not allowed state: " + newStatus;
     	}
     	if (doc.RequestStatus && !isAllowedTransiton(doc.RequestStatus, newStatus)) {
     		// don't update the status just ignore
-    		return "not allowed transition " + doc.RequestStatus + " to " + newStatus;
+    		return "Not allowed transition, from " + doc.RequestStatus + " to " + newStatus;
```

Comment:
I always suggest to put quotes around status'es that it will increase readability especially if status is not integer data-type. For instance, if status is some sentences, e.g. 'status is ok' then reading full string will become more clear that status is shown in quotes.